rct229/rules/section21/section21rule12.py

```diff
@@ -0,0 +1,97 @@
+from rct229.data.schema_enums import schema_enums
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.ruleset_functions.get_baseline_system_types import get_baseline_system_types
+from rct229.utils.assertions import getattr_
+
+VARIABLE_FLOW = schema_enums["FluidLoopFlowControlOptions"].VARIABLE_FLOW
```

Comment:
I'd prefer for us to generally do something like:
```
FLUID_LOOP_FLOW_CONTROL = schema_enums["FluidLoopFlowControlOptions"]
```
and then use FLUID_LOOP_FLOW_CONTROL.VARIABLE_FLOW in the code.
If we do that consistently, I think it will make the intent a little bit clearer.